requirements/requirements-codestyle.txt

```diff
@@ -3,4 +3,4 @@ flake8==2.4.0
 pep8==1.5.7
 
 # Sort and lint imports
-isort==3.9.6
```

Comment:
Probably Windows, as it the problem with the older isort was an encoding problem.